pkg/provisioner/ironic/ironic.go

```diff
@@ -825,6 +825,53 @@ func (p *ironicProvisioner) getUpdateOptsForNode(ironicNode *nodes.Node, data pr
 	return updater
 }
 
+// GetFirmwareSettings gets the BIOS settings and optional schema from the host and returns maps
+func (p *ironicProvisioner) GetFirmwareSettings(includeSchema bool) (settings metal3v1alpha1.SettingsMap, schema map[string]metal3v1alpha1.SettingSchema, err error) {
+
+	if p.nodeID == "" {
+		return nil, nil, provisioner.ErrNeedsRegistration
+	}
+
+	// Get the settings from Ironic via Gophercloud
+	var settingsList []nodes.BIOSSetting
+	var biosListErr error
+	if includeSchema {
+		opts := nodes.ListBIOSSettingsOpts{Detail: true}
```

Comment:
For this to work you need to update the microversion, I think it's in ironic/clients.go

Reply:
Yes, very good point. Changed to use microversion 1.74 and updated docs/configuration.md.